### PR TITLE
feat: multi grid management

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11845](https://app.clickup.com/t/2645698/DEV-11845)

### Description

CleanShot 2024-10-16 at 17 41 36@2x

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11845 Multi Grid management](https://app.clickup.com/t/2645698/DEV-11845)

**Comment by nastita** [comment]
@dzbo When I open my profile this menu button is already open by default 

image

File: components/ModalTemplateSwitchApplicationNetwork.vue [hunk 1]
```diff
@@ -3,16 +3,17 @@ const { closeModal, modal } = useModal()
 const { formatMessage } = useIntl()
 
 const handleChangeNetwork = async () => {
-  const { selectedChainId } = storeToRefs(useAppStore())
   const { getNetworkByChainId } = useAppStore()
   const { disconnect } = useBaseProvider()
 
-  selectedChainId.value = modal?.data?.chainId
   await navigateTo({
```
Comment (richtera) [inline components/ModalTemplateSwitchApplicationNetwork.vue:9]: You can just pass external: true to reload this, no?

**Review by nastita — APPROVED** [review]
super nice 👍, there's small bug when loading the page as I mentioned in a comment

**Review by richtera — APPROVED** [review]
There is a external: true parameter to navigateTo which will also force a reload.

**Comment by dzbo** [comment]
> @dzbo When I open my profile this menu button is already open by default
> 
> image

Thanks @nastita ! I've used this for debugging and forgot to remove 😅

File: components/ModalTemplateSwitchApplicationNetwork.vue (hunk 1 above)
Comment (dzbo) [reply components/ModalTemplateSwitchApplicationNetwork.vue:9]: Ah yes, I forgot about this

**Pull request merged**